pwa/src/templates/componentDetail/ComponentsDetailTemplate.tsx

```diff
@@ -96,8 +96,22 @@ export const ComponentsDetailTemplate: React.FC<ComponentsDetailTemplateProps> =
                   <TableCell>Rhoncus aenean vel elit scelerisque mauris</TableCell>
                 </TableRow>
                 <TableRow>
-                  <TableHeader>UPL</TableHeader>
-                  <TableCell>Sit amet nisl suscipit adipiscing bibendum est ultricies</TableCell>
+                  <TableHeader>{t("Products")}</TableHeader>
+                  <TableCell>
+                    {_getComponent.data.embedded.nl.upl.map((product: string, idx: any) => (
+                      <a
+                        key={idx}
+                        href="http://standaarden.overheid.nl/owms/terms/AangifteVertrekBuitenland"
+                        target={"_blank"}
```

Comment:
Small detail: not relevant once you've updated it to underneath feedback, but: a string prop doesn't require the `{}` syntax, so: `target="_blank"` would work as well.